assets/js/map.js

```diff
@@ -416,6 +416,9 @@ var MapBase = {
 
     var warningText = Cycles.isSameAsYesterday(marker.category) ? `<span class="marker-warning-wrapper"><div><img class="warning-icon" src="./assets/images/same-cycle-alert.png" alt="Alert"></div><p>${Language.get("map.same_cycle_yesterday")}</p></span>` : '';
 
+    if (marker.day == Settings.cycleForUnknownCycles)
+      warningText = `<span class="marker-warning-wrapper"><div><img class="warning-icon" src="./assets/images/same-cycle-alert.png" alt="Alert"></div><p>This item is in unknown cycle, if you find something here, report it on: <a href="https://github.com/jeanropke/RDR2CollectorsMap/issues" target="_blank">GitHub</a> or <a href="https://discord.gg/WWru8cP" target="_blank">Discord</a>.</p></span>`;
```

Comment:
Needs translation. Could you add it? Also, links shouldn't be translatable so just use .replace('{discord}', link) for those please. 😄

Reply:
fixed